lua/autorun/wire_load.lua

```diff
@@ -87,6 +87,7 @@ if SERVER then
 	include("wire/server/modelplug.lua")
 	include("wire/server/debuggerlib.lua")
 	include("wire/server/sents_registry.lua")
+	include("wire/server/wire_map_interface.lua")
```

Comment:
Could be nice if this only gets included when a map interface is present

Reply:
Would cause issues with:
https://github.com/wiremod/wire/pull/3351/files#diff-11f733e8c74f850d57c2ed245e889bf04028edfa9f155d3c91bb44550d46c27cR860-R869

I could add a check if it was loaded to it, but I am not sure if this would remove the overhead you seem to be concerned about. It could also cause issues with save games loaded from the menu, as the `wire_map_interface.lua` would be loaded in too late to matter for that save file.

Please keep in mind that this file in question has already been reviewed once by @thegrb93 and has been optimized in accordance to his suggestions.

Reply:
> wire_map_interface.lua has like no overhead though, and isn't networked, so what's the point? seems like it would cause more headache than its worth to not include the file

yet more gc lag, large servers get already destroyed by wiremods gc cluttering